### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by jglick**
Minor cleanups after the last release. @reviewbybees

### CODE REVIEW

File: src/main/java/hudson/matrix/AxisDescriptor.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public boolean isInstantiable() {
     /**
      * Makes sure that the given name is good as a axis name.
      *
-     * Aside from {@link Jenkins#checkGoodName()} this disallows ',' and
+     * Aside from {@link Jenkins#checkGoodName} this disallows ',' and
```
Comment (oleg-nenashev) [inline src/main/java/hudson/matrix/AxisDescriptor.java]: both options should work btw

Comment (oleg-nenashev) [inline src/main/java/hudson/matrix/AxisDescriptor.java]: Please ignore, Java8 is too strict :(

**Comment by oleg-nenashev** [comment]
:+1:

File: src/main/java/hudson/matrix/AxisDescriptor.java (hunk 1 above)
Comment (jglick) [inline src/main/java/hudson/matrix/AxisDescriptor.java]: The original code was plain wrong: the method takes a `String` parameter. You can either specify the full list of parameter types, or (if there are no overloads) not specify the parameter types.

File: src/main/java/hudson/matrix/MatrixProject.java [hunk 2]
```diff
@@ -216,7 +214,7 @@ public String getPronoun() {
     /**
      * Gets the workspace location that {@link MatrixConfiguration} uses.
      *
-     * @see MatrixRun.MatrixRunExecution#decideWorkspace(Node, WorkspaceList)
+     * Used from {@code MatrixRun.MatrixRunExecution.decideWorkspace}.
```
Comment (tfennelly) [inline src/main/java/hudson/matrix/MatrixProject.java]: Why not `{@link}` it?

**Comment by tfennelly** [comment]
:+1: assuming the build failure has nothing to do with this change.

**Comment by olivergondza** [comment]
Tests hang at some point but that is not related to this PR. It seems it uses all PermGen and choke on GC for a while before it dies with `OutOfMemoryError: PermGen space`.

**Comment by olivergondza** [comment]
Perm gen issue should be addressed in https://github.com/jenkinsci/matrix-project-plugin/pull/21.

File: src/main/java/hudson/matrix/MatrixProject.java (hunk 2 above)
Comment (jglick) [inline src/main/java/hudson/matrix/MatrixProject.java]: Because the class is `private`.

**Pull request merged**